documentation/modules/exploit/windows/http/ws_ftp_rce_cve_2023_40044.md

```diff
@@ -0,0 +1,146 @@
+## Vulnerable Application
+This module exploits an unsafe .NET deserialization vulnerability to achieve unauthenticated remote code
+execution against a vulnerable WS_FTP server running the Ad Hoc Transfer module. All versions of WS_FTP server
+prior to 8.7.4 and 8.8.2 are vulnerable to this issue. The vulnerability was originally discovered by AssetNote.
+
+For a full technical analysis of the vulnerability read the
+[Rapid7 AttackerKB Analysis](https://attackerkb.com/topics/bn32f9sNax/cve-2023-40044/rapid7-analysis).
+
+## Testing
+Download and install a vulnerable version of WS_FTP server. By default the server will listen for HTTPS connections
+on port 443. The vulnerability is in the Ad Hoc Transfer module which is installed by default. This exploit
+module was tested against `2020.0.1`. and `2022.0.1`.
```

Comment:
Please use coherent version schemes: in "Vulnerable Application", "8.7.4 and 8.8.2" is mentioned, yet in "Testing", it's "2020.0.1. and 2022.0.1".

Reply:
Good catch, thanks. I have addressed this in [1695a12](https://github.com/rapid7/metasploit-framework/pull/18414/commits/1695a12c9cdc5a5038fb9ebcb929e0a9fc7242b1).